tests/backends/base/test_base.py

```diff
@@ -55,6 +61,55 @@ def test_check_database_version_supported_with_none_as_database_version(self):
             connection.check_database_version_supported()
 
 
+class DatabaseWrapperLoggingTests(TransactionTestCase):
+    available_apps = []
+
+    @override_settings(DEBUG=True)
+    def test_commit_debug_log(self):
+        conn = connections[DEFAULT_DB_ALIAS]
+        with CaptureQueriesContext(conn):
+            with self.assertLogs("django.db.backends.base", "DEBUG") as cm:
+                with transaction.atomic():
+                    Person.objects.create(first_name="first", last_name="last")
+                    Person.objects.create(first_name="first", last_name="last2")
```

Comment:
Creating two objects should not be necessary.

```suggestion
```

Reply:
removed.